test/unit/distributed_tracing/dt-cats.test.js

```diff
@@ -126,13 +127,13 @@ tap.test('distributed tracing', function (t) {
 
             const attributes = event[0]
             arbitrary.forEach((key) => {
-              t.ok(attributes[`${key}`], `${type} should have ${key}`)
+              assert.ok(attributes[`${key}`], `${type} should have ${key}`)
             })
             unexpected.forEach((key) => {
-              t.notOk(attributes[`${key}`], `${type} should not have ${key}`)
+              assert.ok(!attributes[`${key}`], `${type} should not have ${key}`)
```

Comment:
`assert.notOk` is also available. I would find that easier to read than `!something`.

Reply:
I don't see it in this list (https://nodejs.org/api/assert.html) and whenever I use it it says that assert.notOk is not a function. Maybe I have a different version?

Reply:
Sorry. I must be thinking of `notEqual`.